Pod/Classes/Keyboard/Keyboard.swift

```diff
@@ -28,7 +28,7 @@ public class Keyboard {
      - parameter animated: Whether or not to animate changes in the handler block alongside the keyboard frame changes.
      - parameter handler:  A block in which to perform view changes.
      */
-    public static func addFrameObserver(_ observer: AnyObject, withAnimations animated: Bool = true, handler: FrameChangeHandler) {
+    public static func addFrameObserver(_ observer: AnyObject, withAnimations animated: Bool = true, handler: @escaping FrameChangeHandler) {
```

Comment:
#? Why the change here? Does Swift 4.1 reject this, but earlier versions didn't? Regardless, no worries about semver, since we're either not using it for Swiftilities, or we're below 1.0 so it doesn't matter. Either way, breaking changes are fine.